packages/payments-plugin/src/mollie/mollie.plugin.ts

```diff
@@ -125,9 +126,7 @@ export interface MolliePluginOptions {
 @VendurePlugin({
     imports: [PluginCommonModule],
     controllers: [MollieController],
-    providers: [
-        MollieService,
-        { provide: PLUGIN_INIT_OPTIONS, useFactory: () => MolliePlugin.options }],
+    providers: [MollieService, { provide: PLUGIN_INIT_OPTIONS, useFactory: () => MolliePlugin.options }],
```

Comment:
New formatting rules? I don't think I changed this

Reply:
I don't think so? Anyway, not a big deal.